index.html

```diff
@@ -80,6 +80,13 @@ <h2>About Us</h2>
 </section>
 <hr />
 
+<section id="society-rules">
+  <h2>Society Rules</h2>
+
+  <p>Read about the Rules of Ruby New Zealand Incorporated from <a href="society-rules.pdf" target="_blank">here</a>.</p>
```

Comment:
Preferably the link text would provide sufficient information to describe the resource without additional context this helps screen readers and assistive tech. Perhaps the whole line of text could be moved into the link.